src/linter-jscs.js

```diff
@@ -209,6 +209,31 @@ export default class LinterJSCS {
       return;
     }
 
+    const JSCS = require('jscs');
```

Comment:
Since both `linter` and this are firing on the `onDidSave` event, it makes sense to ensure that `JSCS` is required here, good catch :wink:.